src/main/java/frc/robot/commands/auto/DriveAndScoreConeCommand.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.auto;
+
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants.ArmConstants;
+import frc.robot.commands.DriveStraightCommand;
+import frc.robot.commands.DunkCommand;
+import frc.robot.commands.GrabberCommandForward;
+import frc.robot.commands.RuntopositionCommand;
+import frc.robot.subsystems.ArmSubsystem;
+import frc.robot.subsystems.BlinkinSubsystem;
+import frc.robot.subsystems.DrivetrainSubsystem;
+import frc.robot.subsystems.GrabberSubsystem;
+import frc.robot.subsystems.NavXGyroSubsystem;
+
+/** An example command that uses an example subsystem. */
+public class DriveAndScoreConeCommand extends SequentialCommandGroup {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+
+  /**
+   * Creates a new ExampleCommand.
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public DriveAndScoreConeCommand(NavXGyroSubsystem gyro, DrivetrainSubsystem drive, BlinkinSubsystem blinkin, ArmSubsystem arm, GrabberSubsystem grabber) {
+    super(
+      // TODO: Make this drive distance and/or line up with the cone before the RunToPosition command should run
+      new RuntopositionCommand(arm, ArmConstants.kArmMediumPos, .25).withTimeout(2),
+      new ParallelRaceGroup(new GrabberCommandForward(grabber).withTimeout(.75), new RuntopositionCommand(arm, ArmConstants.kArmMediumPos, .25)),
+      new ParallelRaceGroup(new DriveStraightCommand(gyro, drive, blinkin, .5, .5, 1), new RuntopositionCommand(arm, ArmConstants.kArmStoredPos, .2)).withTimeout(2)
```

Comment:
I think the Drive command in `DriveAndScoreCube` is the key one to have withTimeout. 

In `ParallelRaceGroup`, the first command to timeout ends the whole group, so the logic there is close

In an ideal case (in the future), the Command's `isFinished` method should be used to indicate that the command has reached its target so that there does not need to be a timeout. However, in the case of the `Runtoposition` command, it must keep running in order keep the arm at that position (although this ought to be done in the ArmSubsystem's `periodic` method.)